meta/main.yml

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: MIT
+---
+galaxy_info:
+  author: Pino Toscano <ptoscano@redhat.com>
+  description: Configure connectivity of systems to Red Hat
+  company: Red Hat, Inc.
+
+  license: MIT
+
+  min_ansible_version: "2.9"
+
+  platforms:
+    - name: Fedora
+      versions:
+        - all
+    - name: EL
+      versions:
+        - 9
```

Comment:
Should the rhc role support rhel 7 or 8?

Reply:
at least rhel 8 yes, maybe rhel 7; i want to start small for now, and add them later on